qubesadmin/tools/qvm_start_daemon.py

```diff
@@ -181,9 +267,14 @@ def common_guid_args(self, vm):
 
     @staticmethod
     def guid_pidfile(xid):
-        """Helper function to construct a pidfile path"""
+        """Helper function to construct a GUI pidfile path"""
         return '/var/run/qubes/guid-running.{}'.format(xid)
 
+    @staticmethod
+    def pacat_pidfile(xid):
+        """Helper function to construct an AUDIO pidfile path"""
+        return '/var/run/qubes/pacat-running.{}'.format(xid)
```

Comment:
`pacat-simple-vchan` doesn't create pid file at all. That actually may be something to be fixed, but then I'd name the file just `pacat.XID` (no `-running`).

Reply:
In https://github.com/QubesOS/qubes-gui-daemon/pull/43 I implemented pidfile support. The path is `/var/run/qubes/pacat.<XID>`, so please adjust it here.